src/components/Integrations/BigQuery/FreeAccount/FreeAccount.test.js

```diff
@@ -16,8 +16,8 @@ describe('Bigquery FreeAccount component', () => {
     );
     expect(getByText('big_query.free_text_summary')).toBeInTheDocument();
     expect(getByText('big_query.free_btn_redirect')).toBeInTheDocument();
-    expect(getByText('big_query.free_ul_item_one')).toBeInTheDocument();
-    expect(getByText('big_query.free_ul_item_two')).toBeInTheDocument();
+    expect(getByText('big_query.free_ul_item_insights')).toBeInTheDocument();
+    expect(getByText('big_query.free_ul_item_filter')).toBeInTheDocument();
```

Comment:
Quizás para la comprobación de esos textos, sea mejor usar un arreglo, son muchos textos los que hay que comprobar y son las mismas aserciones las que se hacen. 

Pudiese ser mas sencillo crear un array con los textos que se van a comprobar si están en el documento, algo como: 
```
const texts = [
'big_query.free_ul_item_one',
'big_query.free_btn_redirect',
'big_query.free_ul_item_insights',
'big_query.free_ul_item_filter',
...etc
]
```

Luego iterar: 
`
texts.map(text => expect(getByText(text)).toBeInTheDocument())
`
Como ventaja veo que se reduce un poco el código y se evita tener que copiar siempre el mismo comportamiento de la aserción. También favorece la mantenibilidad

Reply:
suggest accepted and applied